glide.yaml

```diff
@@ -46,6 +46,8 @@ import:
   version: kubernetes-1.12.4
 - package: k8s.io/apiextensions-apiserver
   version: kubernetes-1.12.4
+- package: k8s.io/apiserver
+  version: kubernetes-1.12.4
```

Comment:
Do we need this just for the audit subpackage?  If so, I wonder if we can get away with only sourcing that subpackage rather than all of the apiserver code.  It's okay if we can't.

Reply:
Yes, we need this for audit subpackage and I don't think it's possible to get the subpackage without api server code. https://k8s.io/apiserver/pkg/apis/audit/v1?go-get=1 returns  https://github.com/kubernetes/apiserver as the source for the package.